src/earn/EarnEnterAmount.tsx

```diff
@@ -735,6 +750,89 @@ function FeeDetailsBottomSheet({
   )
 }
 
+function SwapDetailsBottomSheet({
+  forwardedRef,
+  testID,
+  swapTransaction,
+  pool,
+  token,
+}: {
+  forwardedRef: React.RefObject<BottomSheetModalRefType>
+  testID: string
+  swapTransaction: SwapTransaction
+  pool: EarnPosition
+  token: TokenBalance
+}) {
+  const { t } = useTranslation()
+  const depositToken = useTokenInfo(pool.dataProps.depositTokenId)
+
+  if (!depositToken) {
+    // should never happen
+    throw new Error(`Token info not found for token ID ${pool.dataProps.depositTokenId}`)
+  }
+
+  const swapToAmount = useMemo(() => {
+    return new BigNumber(swapTransaction.buyAmount).shiftedBy(-depositToken.decimals).toString()
+  }, [depositToken])
+
+  const swapFromAmount = useMemo(() => {
+    return new BigNumber(swapTransaction.sellAmount).shiftedBy(-token.decimals).toString()
+  }, [token])
```

Comment:
use `swapToAmountInDecimals` and the user inputted amount for swapTo and swapFrom amounts respectively to be consistent with the other places.